iiod_usbd.init

```diff
@@ -69,8 +69,12 @@ case "$1" in
 
 		if start-stop-daemon -S -b -q -m -p /var/run/iiod.pid -x /usr/sbin/iiod -- $IIOD_OPTS; then
 			if [ $START_UDC -eq 1 ] ; then
-				sleep 0.1
-				echo $UDC_NAME > ${GADGET}/UDC
+				for _ in $(seq 1 60) ; do
```

Comment:
does 6 retries ... this more looks like 60

Reply:
oops;
forgot that

Reply:
should we do 10?

Reply:
sounds good

Reply:
bumped to 10

Reply:
wait; wrong branch

Reply:
should be good now